tasklist/webapp/pom.xml

```diff
@@ -540,10 +548,12 @@
         </configuration>
       </plugin>
 
-      <!-- LICENSE PLUGIN -->
       <plugin>
-        <groupId>com.mycila</groupId>
-        <artifactId>license-maven-plugin</artifactId>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
+        <configuration>
+          <excludes>**/GetCustomSnapshotResponse.java</excludes>
```

Comment:
❓ Why we need this?

Reply:
Good point, it is been there since a [while](https://github.com/camunda/tasklist/commit/c5e9c61eb6218da5a04bc0546ef0727029e0232c), I will try to remove it